config/envs/default.py

```diff
@@ -8,7 +8,7 @@
 
 @configclass
 class DefaultConfig(object):
-    FLASK_ROOT = str(Path(__file__).parent.parent.parent)
+    FLASK_ROOT = Path(__file__).parent.parent.parent
```

Comment:
Out of curiosity, why did you remove the type casting to string here?

Reply:
I did this because the casting breaks building the test data paths in the way I copied from the data-store repo. I'm not sure if the string casting is needed because the path object seems to have the same operators as string for concatination etc. But willing to change back if necessary 👍

Reply:
For authentication, we use e.g.

```
DefaultConfig.FLASK_ROOT + "/tests/keys/rsa256/public.pem"
```
in order to set certain paths (this is how we're going to specify the public/private key paths for authentication in submit as well).  Without being cast to string, this will lead to a TypeError as we can't concatenate a PosixPath and a string. 

We can either create two variables (one cast to string and one not), or cast to str where needed. Perhaps leave it as you've done and in my PR for authentication (where we'll need FLASK_ROOT as a str) I'll see where the conflict arises and solve it when we've got both use cases in front of us.